cocos/platform/openharmony/modules/CCCanvasRenderingContext2D-openharmony.cpp

```diff
@@ -245,13 +227,35 @@ class CanvasRenderingContext2DImpl {
                     bool /* smallCaps */) {
         _fontName = fontName;
         _fontSize = static_cast<int>(fontSize);
-        std::string fontPath;
-        if (!_fontName.empty()) {
-            const char* fontFamilies[1];
-            fontFamilies[0] = fontName.c_str();
-            OH_Drawing_SetTextStyleFontFamilies(_textStyle, 1, fontFamilies);
-            OH_Drawing_SetTextStyleLocale(_textStyle, "en");
+        
+        if(_fontCollection) {
+            OH_Drawing_DestroyFontCollection(_fontCollection);
+        }
+        if(_typographyCreate) {
+            OH_Drawing_DestroyTypographyHandler(_typographyCreate);
+        }
+        if(_textStyle) {
```

Comment:
Does _typographyCreate need to be released?

Reply:
The creation of _typographyCreate depends on _fontCollection parameter, so need to be released together.